lms/static/scripts/frontend_apps/components/dashboard/DashboardOrganizationFilters.tsx

```diff
@@ -0,0 +1,111 @@
+import { MultiSelect } from '@hypothesis/frontend-shared';
+
+import type { Assignment, Course, Student } from '../../api-types';
+import { useConfig } from '../../config';
+import { useAPIFetch } from '../../utils/api';
+
+export type DashboardFiltersProps = {
+  selectedCourses: Course[];
+  onCoursesChange: (newCourses: Course[]) => void;
+  selectedAssignments: Assignment[];
+  onAssignmentsChange: (newAssignments: Assignment[]) => void;
+  selectedStudents: Student[];
+  onStudentsChange: (newStudents: Student[]) => void;
+};
+
+export default function DashboardOrganizationFilters({
```

Comment:
The name of this component is a bit confusing because it doesn't filter organizations, rather it is a set of filters displayed in the `OrganizationActivity` view. `OrganizationActivityFilters` would make more sense as a name.

Reply:
Totally